src/components/ChatList/ChannelListHeaderItem.tsx

```diff
@@ -4,7 +4,7 @@ import {StyleSheet, Text, View} from 'react-native';
 
 import dimen from '../../utils/dimen';
 import {COLORS} from '../../utils/theme';
-import {SIGNED} from '../../hooks/core/constant';
+import {ANONYMOUS, SIGNED} from '../../hooks/core/constant';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been modified to include `ANONYMOUS` from the constants file. This change seems necessary for the new logic introduced in line 57 where the type of user is checked to determine how the name should be displayed.
```diff
- import {SIGNED} from '../../hooks/core/constant';
+ import {ANONYMOUS, SIGNED} from '../../hooks/core/constant';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->